yjit.rb

```diff
@@ -146,14 +146,17 @@ def self.runtime_stats
     # Average length of instruction sequences executed by YJIT
     avg_len_in_yjit = retired_in_yjit.to_f / total_exits
 
-    # Proportion of instructions that retire in YJIT
-    total_insns_count = retired_in_yjit + stats[:vm_insns_count]
-    yjit_ratio_pct = 100.0 * retired_in_yjit.to_f / total_insns_count
+    # This only available on yjit stats builds
+    if stats.has_key?(:vm_insns_count)
```

Comment:
```suggestion
    if stats.key?(:vm_insns_count)
```

`has_key?` has been (softly) deprecated by Matz since 10 years ago https://web.archive.org/web/20210512014514/http://blade.nagaokaut.ac.jp/cgi-bin/scat.rb/ruby/ruby-core/43765. It's unlikely that we'll remove this soon, but it's more likely to go away than `key?`, at least.

Reply:
Good to know. When I googled it that's what came up